src/components/widgetCRUD/repo/widgstConfFSRepo.ts

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 Cognite AS
```

Comment:
please add comments to this and previous one explaining the difference.

Reply:
done